### PR TITLE
Draft PR: CA Judges Import

### PR DESCRIPTION
**Pull request by litewarp**

### CODE REVIEW

File: cl/people_db/models.py [hunk 1]
```diff
@@ -626,22 +626,28 @@ class Position(AbstractDateTimeModel):
         ("rec_postpone", "Recommendation Postponed"),
         ("rec_bad", "Recommended Unfavorably"),
     )
+    ELECTION_PARTISAN = "e_part"
+    ELECTION_NON_PARTISAN = "e_non_part"
+    APPOINTMENT_PRESEDENT = "a_pres"
```
Comment (mlissner) [inline cl/people_db/models.py:631]: Sorry, I know this is my mistake, but can you fix this typo before it becomes part of the legacy we leave our children?

File: cl/people_db/import_judges/ca_judges_import_helpers.py [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+    appellate_courts = {
+        "First": "1",
+        "Second": "2",
+        "Third": "3",
+        "Fourth": "4",
+        "Fifth": "5",
+        "Sixth": "6",
+    }
+    lookup = appellate_courts[fourth_word]
+    return lookup
```
Comment (mlissner) [inline cl/people_db/import_judges/ca_judges_import_helpers.py:297]: This could just be a dict. :shrug:

**Review by mlissner — COMMENTED** [review]
I only had time for a skim, but here are a couple little things. It's looking a lot better. Thanks for all the cleanup.

File: cl/people_db/import_judges/ca_judges_import_helpers.py (hunk 2 above)
Comment (litewarp) [reply cl/people_db/import_judges/ca_judges_import_helpers.py:297]: Ugh, fine.

**Comment by litewarp** [comment]
This PR is getting a bit messy, so here's the update on what's left: 

It's basically done except for debugging and one more function I need to write and that's pre-processing the individual positions based on information from all of them. This includes:
 
1. Ascertaining the termination_date and reason for each position by looking forward or backward through the listed positions.

2. Making sure that the dates don't overlap. Currently some of the positions are concurrently listed. For example, if a person is a Superior Court Judge between 2001-2005 but they are elected as the Presiding Judge from 2003-2004, there will be two overlapping entries. 

Unless you're ok with the duplication, I think the solution here would be to create a position from 2001-2003 as "Judge" then a position from "2003-2004" for Presiding Judge, and then another one from 2004-2005 as "Judge"

3. Handling the edge cases (see lines 289-337 in the "ca_judges_import_helpers" file).

**Comment by mlissner** [comment]
> Unless you're ok with the duplication, I think the solution here would be to create a position from 2001-2003 as "Judge" then a position from "2003-2004" for Presiding Judge, and then another one from 2004-2005 as "Judge"

Yep, that sounds right to me.

**Pull request closed without merging**